### ISSUE TITLE
Internal Server Error when swagger is disabled and viewed

### ISSUE THREAD

**Issue by mariuszs** [4finance/micro-infra-spring#92]
Disabled with  `-Dcom.ofg.infrastructure.swagger.auto=false`  and opened http://localhost:8095/swagger

```
There was an unexpected error (type=Internal Server Error, status=500).
${message}
```

**Comment by mchmielarz** [comment]
Upgrading of spring-boot to 1.2.1 and using the latest version of micro-infra-spring solved the issue with displaying `/swagger/index.html`.

Thymeleaf breaks view on `/swagger` url (see #21).

**Comment by szpak** [comment]
So, it will be fixed when #167 is merged.
